apps/cookbook/src/app/examples/button-example/examples/_grid-layout.scss

```diff
@@ -0,0 +1,8 @@
+@use '@kirbydesign/core/src/scss/utils';
+
+:host {
+  display: inline-grid;
+  grid-template-columns: repeat(4, auto);
+  place-items: center center;
+  gap: utils.size('s');
```

Comment:
Consider using flexbox with wrapping for this instead. That could prevent the buttons from overlapping at certain viewport widths (e.g. around 1000px wide).

Reply:
Fixed